### DISCUSSION TITLE
Date is not showing correct some times on page

### DISCUSSION THREAD

**Discussion by Priy16** [microsoft-search/pnp-modern-search#1539] [Q&A]
Hi Team,

We have page that have some columns and that column data coming from a list with help of search crawl.
There is a UP To Single line text column on page, that showing date from managed metadata column of the list, but UP To column date 

And after again refreshing the page showing the correct but same time some other record which have correct date showing incorrect.

For more reference attaching some files.
some times showing incorrect after ![UptoColumnIssue](https://user-images.githubusercontent.com/94505069/142134293-ce1a9542-f785-4d0a-8792-e24d694d243c.jpg)the page of any record.

![UptoColumnIssueCorrectOne](https://user-images.githubusercontent.com/94505069/142134365-c238d13e-6330-48ed-8da3-039fb3b23de2.jpg)

**Comment by wobba** [comment]
Can you expand on which version you are using of the web parts? And if logging as an issue you need to use the templates and fill out all information regarding configuration steps for repro.

**Comment by Priy16** [comment]
PROTECTED

Hi Mikael,

Greetings.

Thanks a lot for your mail.

Screen shot is of a webpart, it's not of a library. We already had a discussion with this issue with Microsoft.
Microsoft given us GitHub URL to raise this ticket.

Attached Microsoft mail for reference.

Also sharing screen shot of web part this page. Its PnP search web part version 3.
[cid:1bd90831-54d6-48c1-a8d7-6a8e9e160358]

[cid:20785ace-d552-4e55-8f30-dae51b891979]


Feel free to reach out us for more information.

If you need to call regarding this issue, we will discuss it call too.

Awaiting for your valuable response.

**Reply by wobba** [reply]
You need to log exact repro steps in order to triage. So please log this in the Issues tab, and fill out all the steps in the template. To determine if it's a data issue or UX issue you have to look at the search response data as well - to compare with the UX.
